### ISSUE TITLE
Type not recognized at resource response level

### ISSUE THREAD

**Issue by eleonoraortega** [mulesoft-labs/rajapa#21]
Using the following example 

``` yaml
#%RAML 1.0
title: API with Types
types:
  User:
    type: object
    properties:
      firstname: string
      lastname:  string
      age:       number
/users/{id}:
  get:
    responses:
      200:
        body:
          application/json:
            type: User
```

The parser throws   - Unexpected key

**Comment by svacas** [comment]
fixed by 369929472cfa31f01ffcfa5f8ed40dd2c2a9031f

**Comment by eleonoraortega** [comment]
ok
